pandas/tests/reshape/test_melt.py

```diff
@@ -661,3 +661,20 @@ def test_col_substring_of_stubname(self):
                                  i=['node_id', 'A'],
                                  j='time')
         tm.assert_frame_equal(result, expected)
+
+    def test_melt_missing_columns(self):
+        # Addresses issue #23575
```

Comment:
We usually write this as `GH-23575` or provide the full GitHub URL to the issue.